### PR TITLE
Update Bitcanna to v1.6.3 and Band to v2.5.1

### PR DESCRIPTION
**Pull request by rodri-r**

### CODE REVIEW

**Comment by andy108369** [comment]
@tombeynon FYI: this is @rodri-r 's first PR.
I'll try to check it on Monday

File: bitcanna/build.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ services:
...
+        VERSION: v1.6.2
         REPOSITORY: https://github.com/BitCannaGlobal/bcna
         NAMESPACE: BCNAD
     ports:
       - '26656:26656'
       - '26657:26657'
       - '1317:1317'
     environment:
-      - MONIKER=node_1
+      - MONIKER=R-Node
```
Comment (andy108369) [inline bitcanna/build.yml:19]: please keep moniker as `node_1` just like the rest of the projects.

File: bitcanna/deploy.yml [hunk 2]
```diff
@@ -39,11 +39,6 @@ profiles:
           #     persistent: true
   placement:
     dcloud:
-      attributes:
-        host: akash
-      signedBy:
-        anyOf:
-          - akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63
```
Comment (andy108369) [inline bitcanna/deploy.yml:46]: please keep these.
each project has them set as this helps to default the deployments to tested/curated providers.

File: bitcanna/docker-compose.yml [hunk 3]
```diff
@@ -2,17 +2,16 @@ version: '3.4'
...
-    image: ghcr.io/akash-network/cosmos-omnibus:v0.3.26-bitcanna-v1.5.3
+    image: ghcr.io/akash-network/cosmos-omnibus:v0.3.26-generic
     ports:
       - '26656:26656'
       - '26657:26657'
       - '1317:1317'
     environment:
-      - MONIKER=node_1
+      - BINARY_URL=https://github.com/BitCannaGlobal/bcna/releases/download/v1.6.2/bcna_linux_amd64.tar.gz
+      - MONIKER=R-Drone
```
Comment (andy108369) [inline bitcanna/docker-compose.yml:12]: please keep moniker as `node_1` just like the rest of the projects.

File: bitcanna/docker-compose.yml [hunk 4]
```diff
@@ -2,17 +2,16 @@ version: '3.4'
...
       - CHAIN_JSON=https://raw.githubusercontent.com/cosmos/chain-registry/master/bitcanna/chain.json
       - P2P_POLKACHU=1
       - STATESYNC_POLKACHU=1
-    env_file:
-      - ../.env
```
Comment (andy108369) [inline bitcanna/docker-compose.yml:16]: please keep this. The rest of the projects use that for specifying the S3 creds.

**Review by andy108369 — CHANGES_REQUESTED** [review]
I think all you need is just to bump the version numbers. :-)

1. Please address to the requested changes (comments)
2. Please bump bitcanna version in the `.github/workflows/publish.yml` file as well.

Let me know if you have any questions / or need any help!

**Comment by rodri-r** [comment]
Thank you Andy, I have made the requested changes.  

I thought that when the PR got merged and the new release cut, actions would automatically grab my image and rename things to ghcr.io/akash-network/

**Comment by tombeynon** [comment]
Hey @rodri-r thanks so much for the PR. As per Andrey's comments, really we just want to change the version number and nothing else. Check the 'Files Changed' tab on Github to make sure nothing else has slipped through, there are still a few other changes.

Ultimately the process for a chain upgrade is just to find and replace the version number in most cases. Sometimes GOLANG_VERSION needs to be updated in build.yml if a newer Go version is required (default is 1.18 currently).

**Comment by rodri-r** [comment]
Thank you @tombeynon I understand now, I will make the changes.

**Comment by tombeynon** [comment]
Awesome thanks @rodri-r! There should be a couple more files, mainly `.github/workflows/publish.yml` and the main project `README.md`. Can you search for `v1.5.3` and see if there are any more matches?

**Review by rodri-r — COMMENTED** [review]
Upgraded to v1.6.3

**Comment by rodri-r** [comment]
Thanks @tombeynon I think I have it now in the new upgrade to v1.6.3

**Comment by tombeynon** [comment]
Lovely thanks for going straight to 1.6.3. Just README.md in the project root is missing now

**Comment by rodri-r** [comment]
Damn, forgot that Readme. :s 
I've made changes, think it is good now.  Next PR, I will try to send upgrades in batches. Thank you again.

**Comment by andy108369** [comment]
> Next PR, I will try to send upgrades in batches

No need doing that in batches :D
Ideal PR is to make one logical change in a single PR.

And yours looks great now! :-)

**Review by andy108369 — APPROVED** [review]

**Pull request merged**